2019-04-03 02:34:41.2953 | 解压地址:ID=1 
2019-04-03 02:34:54.2813 | 解压地址:ID=1 
2019-04-03 02:34:57.1367 | 解压地址:ID=1 
2019-04-03 17:34:02.5880 | Connection id "0HLLO2U0VK9QV", Request id "0HLLO2U0VK9QV:00000001": An unhandled exception was thrown by the application. Castle.MicroKernel.ComponentNotFoundException: No component for supporting the service ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware was found
   at Castle.MicroKernel.DefaultKernel.Castle.MicroKernel.IKernelInternal.Resolve(Type service, IDictionary arguments, IReleasePolicy policy)
   at Castle.Windsor.MsDependencyInjection.ScopedWindsorServiceProvider.GetServiceInternal(Type serviceType, Boolean isOptional) in D:\Github\castle-windsor-ms-adapter\src\Castle.Windsor.MsDependencyInjection\ScopedWindsorServiceProvider.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.AspNetCore.Http.MiddlewareFactory.Create(Type middlewareType)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 17:34:12.5346 | Connection id "0HLLO2U0VK9QV", Request id "0HLLO2U0VK9QV:00000002": An unhandled exception was thrown by the application. Castle.MicroKernel.ComponentNotFoundException: No component for supporting the service ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware was found
   at Castle.MicroKernel.DefaultKernel.Castle.MicroKernel.IKernelInternal.Resolve(Type service, IDictionary arguments, IReleasePolicy policy)
   at Castle.Windsor.MsDependencyInjection.ScopedWindsorServiceProvider.GetServiceInternal(Type serviceType, Boolean isOptional) in D:\Github\castle-windsor-ms-adapter\src\Castle.Windsor.MsDependencyInjection\ScopedWindsorServiceProvider.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.AspNetCore.Http.MiddlewareFactory.Create(Type middlewareType)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 17:34:14.6307 | Connection id "0HLLO2U0VK9QV", Request id "0HLLO2U0VK9QV:00000003": An unhandled exception was thrown by the application. Castle.MicroKernel.ComponentNotFoundException: No component for supporting the service ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware was found
   at Castle.MicroKernel.DefaultKernel.Castle.MicroKernel.IKernelInternal.Resolve(Type service, IDictionary arguments, IReleasePolicy policy)
   at Castle.Windsor.MsDependencyInjection.ScopedWindsorServiceProvider.GetServiceInternal(Type serviceType, Boolean isOptional) in D:\Github\castle-windsor-ms-adapter\src\Castle.Windsor.MsDependencyInjection\ScopedWindsorServiceProvider.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.AspNetCore.Http.MiddlewareFactory.Create(Type middlewareType)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 17:42:58.2395 | Connection id "0HLLO330KF7J4", Request id "0HLLO330KF7J4:00000001": An unhandled exception was thrown by the application. Castle.MicroKernel.ComponentNotFoundException: No component for supporting the service ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware was found
   at Castle.MicroKernel.DefaultKernel.Castle.MicroKernel.IKernelInternal.Resolve(Type service, IDictionary arguments, IReleasePolicy policy)
   at Castle.Windsor.MsDependencyInjection.ScopedWindsorServiceProvider.GetServiceInternal(Type serviceType, Boolean isOptional) in D:\Github\castle-windsor-ms-adapter\src\Castle.Windsor.MsDependencyInjection\ScopedWindsorServiceProvider.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.AspNetCore.Http.MiddlewareFactory.Create(Type middlewareType)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 17:42:59.3747 | Connection id "0HLLO330KF7J4", Request id "0HLLO330KF7J4:00000002": An unhandled exception was thrown by the application. Castle.MicroKernel.ComponentNotFoundException: No component for supporting the service ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware was found
   at Castle.MicroKernel.DefaultKernel.Castle.MicroKernel.IKernelInternal.Resolve(Type service, IDictionary arguments, IReleasePolicy policy)
   at Castle.Windsor.MsDependencyInjection.ScopedWindsorServiceProvider.GetServiceInternal(Type serviceType, Boolean isOptional) in D:\Github\castle-windsor-ms-adapter\src\Castle.Windsor.MsDependencyInjection\ScopedWindsorServiceProvider.cs:line 55
   at Microsoft.Extensions.DependencyInjection.ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
   at Microsoft.AspNetCore.Http.MiddlewareFactory.Create(Type middlewareType)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 17:49:42.6254 | An unhandled exception has occurred while executing the request. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 230
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context) 
 AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException 

2019-04-03 17:50:52.2805 | An unhandled exception has occurred while executing the request. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 230
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context) 
 AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException 

2019-04-03 18:05:56.1583 | Connection id "0HLLO3FPAP1DV", Request id "0HLLO3FPAP1DV:00000004": An unhandled exception was thrown by the application. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 231
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:05:58.7572 | Connection id "0HLLO3FPAP1DV", Request id "0HLLO3FPAP1DV:00000005": An unhandled exception was thrown by the application. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 231
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:05:59.4243 | Connection id "0HLLO3FPAP1DV", Request id "0HLLO3FPAP1DV:00000006": An unhandled exception was thrown by the application. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 231
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:05:59.9907 | Connection id "0HLLO3FPAP1DV", Request id "0HLLO3FPAP1DV:00000007": An unhandled exception was thrown by the application. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 231
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:07:02.7447 | Connection id "0HLLO3GF39D6R", Request id "0HLLO3GF39D6R:00000001": An unhandled exception was thrown by the application. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Startup.<>c.<<Configure>b__3_5>d.MoveNext() in G:\projects\ShortUrl\ShortUrl.Api\Startup.cs:line 231
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:07:10.1938 | Exception of type 'System.Exception' was thrown. System.Exception: Exception of type 'System.Exception' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 64
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextExceptionFilterAsync() 
 <InvokeNextResourceFilter>d__23.MoveNext => ResourceInvoker.Next => ExceptionFilter.OnException 

2019-04-03 18:08:54.7367 | Connection id "0HLLO3HELBJ3M", Request id "0HLLO3HELBJ3M:00000001": An unhandled exception was thrown by the application. System.Exception: Exception of type 'System.Exception' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 64
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:09:33.1614 | Connection id "0HLLO3HQTB1DD", Request id "0HLLO3HQTB1DD:00000001": An unhandled exception was thrown by the application. System.Exception: Exception of type 'System.Exception' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 64
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware.InvokeAsync(HttpContext context, RequestDelegate next) in G:\projects\ShortUrl\ShortUrl.Api\App\Middlewares\HandleAppExceptionMiddleware.cs:line 17
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:09:37.7696 | Connection id "0HLLO3HQTB1DD", Request id "0HLLO3HQTB1DD:00000002": An unhandled exception was thrown by the application. System.Exception: Exception of type 'System.Exception' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 64
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at ShortUrl.Api.App.Middlewares.HandleAppExceptionMiddleware.InvokeAsync(HttpContext context, RequestDelegate next) in G:\projects\ShortUrl\ShortUrl.Api\App\Middlewares\HandleAppExceptionMiddleware.cs:line 17
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass5_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpProtocol.ProcessRequests[TContext](IHttpApplication`1 application) 
 ExecutionContext.RunInternal => <ProcessRequests>d__185`1.MoveNext => HttpProtocol.ReportApplicationError 

2019-04-03 18:10:54.1422 | An unhandled exception has occurred while executing the request. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 65
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context) 
 AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException 

2019-04-03 18:43:40.1291 | An unhandled exception has occurred while executing the request. ShortUrl.Api.Exceptions.AppException: Exception of type 'ShortUrl.Api.Exceptions.AppException' was thrown.
   at ShortUrl.Api.Controllers.UrlController.Go(UnzipInputModel input) in G:\projects\ShortUrl\ShortUrl.Api\Controllers\UrlController.cs:line 67
   at lambda_method(Closure , Object , Object[] )
   at Microsoft.Extensions.Internal.ObjectMethodExecutor.Execute(Object target, Object[] parameters)
   at Microsoft.AspNetCore.Mvc.Internal.ActionMethodExecutor.SyncActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeActionMethodAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeNextActionFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.InvokeInnerFilterAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeNextResourceFilter()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeFilterPipelineAsync()
   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.InvokeAsync()
   at Microsoft.AspNetCore.Builder.RouterMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.StaticFiles.StaticFileMiddleware.Invoke(HttpContext context)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.MigrationsEndPointMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.EntityFrameworkCore.DatabaseErrorPageMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context) 
 AsyncMethodBuilderCore.Start => <Invoke>d__7.MoveNext => DiagnosticsLoggerExtensions.UnhandledException 

2019-04-03 19:45:37.6866 | 解压地址:ID=1 
2019-04-03 19:45:37.6866 | 解压地址:ID=1 
2019-04-03 19:46:41.8276 | 解压地址:ID=1 
2019-04-03 19:46:59.2029 | 解压地址:ID=1 
2019-04-03 19:48:33.0765 | 解压地址:ID=1 
2019-04-03 19:48:44.9850 | 解压地址:ID=1 
2019-04-03 19:49:08.4076 | 解压地址:ID=1 
2019-04-03 19:49:24.7672 | 解压地址:ID=1 
2019-04-03 19:50:04.8290 | 解压地址:ID=1 
2019-04-03 19:50:23.7023 | 解压地址:ID=1 
2019-04-03 19:52:21.4834 | 解压地址:ID=1 
2019-04-03 19:52:33.5320 | 解压地址:ID=1 
2019-04-03 19:53:14.0475 | 解压地址:ID=1 
2019-04-03 19:53:53.5764 | 解压地址:ID=1 
2019-04-03 19:54:21.6546 | 解压地址:ID=1 
2019-04-03 19:55:07.2516 | 解压地址:ID=1 
2019-04-03 19:55:35.0479 | 解压地址:ID=1 
2019-04-03 19:56:19.2678 | 解压地址:ID=1 
